app/uk/gov/hmrc/helptosavetestadminfrontend/config/AppConfig.scala

```diff
@@ -47,8 +47,11 @@ class AppConfig @Inject()(val runModeConfiguration: Configuration, environment:
     s"""{
           "client_secret":"$clientSecret",
           "client_id":"$clientId",
-          "grant_type":"authorization_code",
+          "grant_type":"client_credentials",
```

Comment:
for privileged access this is the correct value, but for user-restricted access we still need `authorization_code`

Reply:
ok